### ISSUE TITLE
Dark Souls random inputs

### ISSUE THREAD

**Issue by roxzii** [jakibaki/hid-mitm#13]
In Dark souls, even only having hid-mitm on (pc client off) the inputs are random, but not always the same button, sometimes an attack, estus, or jump, all while running to the left. Even after connecting both the 360 controller through the pc client and android client and trying to use the joycons. The only way to fix this is to reboot, turn off hid-mitm, and reboot again.
 In Hollow Knight couldn't go past the start screen, although the controller appeared as connected on Horizon, no input in game.
 Other games like SMO or BOTW were basically perfect, although the sheikah telescope (normally L3) is mapped to the central Xbox button. That and the previously metioned stick drift where the only detractores of an otherwise perfect experience in SMO and BOTW.

**Comment by LeZfX** [comment]
Yep in dark souls the character keeps running to the left with the shield up. Bump.
